src/controllers/controller.bar.js

```diff
@@ -87,7 +87,14 @@ defaults._set('horizontalBar', {
 
 			label: function(item, data) {
 				var datasetLabel = data.datasets[item.datasetIndex].label || '';
-				return datasetLabel + ': ' + item.xLabel;
+				//  float-bar support, if y arguments are array tooltip will show bottom and up values
+				var Yvalue = data.datasets[item.datasetIndex].data[item.index];
```

Comment:
please call this `yValue` instead as we always start variable names with lowercase letters